release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -4238,4 +4238,29 @@ $ oc adm release info 4.12.40 --pullspecs
 [id="ocp-4-12-40-updating"]
 ==== Updating
 
+To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+
+[id="ocp-4-12-41"]
+=== RHSA-2023:6126 - {product-title} 4.12.41 bug fix and security update
+
+Issued: 2023-11-01
+
+{product-title} release 4.12.41, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:6126[RHSA-2023:6126] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:6128[RHSA-2023:6128] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.12.41 --pullspecs
+----
+
+[id="ocp-4-12-41-bug-fixes"]
+==== Bug fixes
+
+* Previously, a missing developer permission caused a web console performance issue with the `HelmChartRepository` resource. Now, you can view all `HelmChartRepository` resources and receive updates from each resource without experiencing the performance issue. (link:https://issues.redhat.com/browse/OCPBUGS-18309[*OCPBUGS-18309*])
```

Comment:
This bug needs to be removed because of the security level.

Reply:
Well spotted. Removed.